### PR TITLE
Fix/RTENU-84 subcategory page title and breadcrumbs

### PR DESCRIPTION
**Pull request by NinaDang97**
- Remove pageTitle props and replace with Breadcrumbs
- Change subcategory pages' routes --> `/main-category/sub-category`

Screenshot 2023-02-06 at 11 37 10

### CODE REVIEW

File: packages/frontend/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -42,7 +43,7 @@ export const Footer = () => {
         <Grid item mobile={12} tablet={6} desktop={6}>
           <Typography variant="body1" sx={{ marginTop: '24px' }}>
             Väyläviraston julkinen verkkopalvelu
-            <Link href="https://vayla.fi/" style={LinkStyle}>
+            <Link href="https://vayla.fi/" target="_blank" style={LinkStyle}>
```
Comment (mehtis) [inline packages/frontend/src/components/Footer/index.tsx:46]: I'd add `rel="noopener"` when opening to new tabs, even if they are to Väylävirasto. Just good security practice.

File: packages/frontend/src/routes.tsx [hunk 2]
```diff
@@ -30,97 +30,62 @@ import { MonitoringEquipment } from './pages/Others/MonitoringEquipment';
...
 const SEARCH_ROUTE = getProtectedRoute(Routes.SEARCH_RESULT, <SearchResult />);
 
-const SubCategoryNames = getSubCategoryData(categoryData);
-
 const DIAGRAMS_ROUTES = [
-  getProtectedRoute(Routes.LINE_DIAGRAMS, <LineDiagrams />, SubCategoryNames.LINE_DIAGRAMS),
-  getProtectedRoute(Routes.SPEED_DIAGRAMS, <SpeedDiagrams />, SubCategoryNames.SPEED_DIAGRAMS),
-  getProtectedRoute(Routes.TRACK_DIAGRAMS, <TrackDiagrams />, SubCategoryNames.TRACK_DIAGRAMS),
-  getProtectedRoute(Routes.GROUPING_DIAGRAMS, <GroupingDiagrams />, SubCategoryNames.GROUPING_DIAGRAMS),
+  getProtectedRoute(Routes.LINE_DIAGRAMS, <LineDiagrams />, { crum: (data: any) => <span>{data.threadName}</span> }),
```
Comment (mehtis) [inline packages/frontend/src/routes.tsx:49]: Is there going to be crums for other routes?

**Review by mehtis — COMMENTED** [review]
Small comments based on reviewing code. I'd have Santeri test this out as well.

**Review by saneDG — APPROVED** [review]

**Comment by saneDG** [comment]
Tested the frontend app locally. Great job!

**Pull request merged**